migration/migrator/data/course_tables.sql

```diff
@@ -497,6 +497,8 @@ CREATE TABLE users (
     instructor_updated boolean NOT NULL DEFAULT false,
     manual_registration boolean DEFAULT false,
     last_updated timestamp(6) with time zone,
+    time_zone VARCHAR NOT NULL DEFAULT 'NOT_SET/NOT_SET',
+    display_image_state VARCHAR NOT NULL DEFAULT 'system',
```

Comment:
Why is this column being added here?

Reply:
I was trying to sneak it in because its part of the same issue, but you're right it has nothing to do with the PR so I removed all references to it.